components/EventDetail.js

```diff
@@ -24,8 +28,19 @@ const StyledEventInfo = styled.p`
 `;
 const StyledDescription = styled.p``;
 
+const confirmDeleteMessage = "Are you sure you want to delete the event?";
+
 export default function EventDetail({ event }) {
   const { day, month, year, time } = getDate(event.startDateTime);
+  const router = useRouter();
+
+  async function handleDelete() {
+    if (confirm(confirmDeleteMessage) == true) {
+      deleteEvent(event._id);
```

Comment:
add await here:   await deleteEvent(event._id);

Reply:
Thank you for your suggesstion. We fixed!